dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EventDAO.java

```diff
@@ -91,6 +91,17 @@ public void deleteEventByEntryID(long entryId) {
         currentSession().flush();
     }
 
+    public void deleteEventByOrganizationID(long organizationId) {
+        Query query = namedQuery("io.dockstore.webservice.core.Event.deleteByOrganizationId");
+        query.setParameter("organizationId", organizationId);
+        query.executeUpdate();
+        // Flush after executing the DELETE query. This would force Hibernate to synchronize the state of the
+        // current session with the database so the session can see that an organization has been deleted
```

Comment:
You're deleting events here, not an organization.

Reply:
Whoops, was thinking about organizations while writing that comment down. Corrected the keyword to events.